classes/helper/HelperList.php

```diff
@@ -55,7 +55,7 @@ class HelperListCore extends Helper
     /** @var string ORDER BY clause determined by field/arrows in list header */
     public $orderBy;
 
-    /** @var string Default ORDER BY clause when $orderBy is not defined */
+    /** @var string Default ORDER BY clause when orderBy is not defined */
```

Comment:
```suggestion
    /** @var string Default ORDER BY clause when $orderBy is not defined */
```

Reply:
yes, but CS fixer does not like it, what should I do? :)

Reply:
It maybe needs to be escaped

Reply:
but how?

Reply:
with tilde "`"